backend/server/controllers/posts.js

```diff
@@ -404,3 +405,52 @@ export const unsharePostById = async (req, res) => {
         res.status(500).send(error)
     }
 }
+
+export const getInteractedUsers = async (req, res) => {
+    try {
+        // Authentication
+        const authToken = req.get('Authorization')
+
+        if (!authToken) {
+            res.status(400).send({
```

Comment:
Use `return res.status(...)`

Reply:
Probably won't do this for now. Lots of our endpoints return Promise<void> so we'd probably want to change lots of things to have return res.status(...). I think this may be something for another issue.